master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +430,284 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
+  > the reverse direction, there will be no conntrack state for the return
+  > packets to match, and there is no application of any egress rules that may
+  > be defined by the untracked policy - so unless the workload's policy
+  > specifically allows the relevant source IP, the return packet will be
+  > dropped.  That is the same overall result as if there was no untracked
+  > policy at all, so in practice it is as if untracked policies do not apply
+  > to this flow.
+
+For packets that arrive on a host interface and are destined for a local
+server process in the host namespace:
+
+- Untracked, pre-DNAT and normal policies all apply.
+
+- If a packet is explicitly allowed by untracked policy, it skips over any
+  pre-DNAT and normal policy.
+
+- If a packet is explicitly allowed by pre-DNAT policy, it skips over any
+  normal policy.
+
+For packets that arrive on a host interface (A) and are forwarded out of the
+same or another host interface (B):
+
+- Untracked policies apply, for both host interfaces A and B, but only the
+  ingress rules that are defined in those policies.  The forwards direction is
+  governed by the ingress rules of untracked policies that apply to interface
+  A, and the reverse direction is governed by the ingress rules of untracked
+  policies that apply to interface B, so those rules should be defined
+  symmetrically.
+
+- Pre-DNAT policies apply, specifically the ingress rules of the pre-DNAT
+  policies that apply to interface A.  (The reverse direction is allowed by
+  conntrack state.)
+
+- Normal policies apply, specifically the ingress rules of the normal policies
+  that apply to interface A, and the egress rules of the normal policies that
+  apply to interface B.  (The reverse direction is allowed by conntrack state.)
+
+- If a packet is explicitly allowed by untracked policy, it skips over any
+  pre-DNAT and normal policy.
+
+- If a packet is explicitly allowed by pre-DNAT policy, it skips over any
+  normal policy.
+
+For packets that are sent from a local server process (in the host namespace)
+out of a host interface:
+
+- Untracked policies apply, specifically the egress rules of the untracked
+  policies that apply to the host interface.
+
+- Normal policies apply, specifically the egress rules of the normal policies
+  that apply to that host interface.
+
+- Pre-DNAT policies do not apply.
+
+For packets that are sent from a local workload out of a host interface:
+
+- No host endpoint policies apply.
+
+## Pre-DNAT policy: a worked example
+
+Imagine a Kubernetes cluster, that its administrator wants to secure as much as
+possible against incoming traffic from outside the cluster.  Let's suppose that:
+
+- The cluster provides various useful Services that are exposed as Kubernetes
+  NodePorts - i.e. as well-known TCP port numbers that appear to be available
+  on any node in the cluster.
+
+- Most of those Services, however, should not be accessed via _any_ node, but
```

Comment:
Before I read through the examples below I thought you were going to block nodes and pods from being able to access these NodePorts too (if the traffic was not destined for the LoadBalancer IP).  I'd like to see it mentioned that the traffic being blocked is external traffic only.